stable/prometheus/templates/alertmanager-pvc.yaml

```diff
@@ -22,6 +17,13 @@ metadata:
 spec:
   accessModes:
 {{ toYaml .Values.alertmanager.persistentVolume.accessModes | indent 4 }}
+{{- if .Values.alertmanager.persistentVolume.storageClass }}
+{{- if (eq "-" .Values.alertmanager.persistentVolume.storageClass) }}
+  storageClassName: ""
+{{- else }}
+  storageClassName: "{{ .Values.alertmanager.persistentVolume.storageClass }}"
+{{- end }}
+{{- end }}
```

Comment:
OK with me. Plus, now that v1.7 is out, we should formally deprecate v1.5 support and focus on v1.6 and v1.7.